cockroachcloud/quickstart.md

```diff
@@ -1,21 +1,25 @@
 ---
-title: Quickstart with CockroachDB Serverless (beta)
-summary: Learn how to create and use your free CockroachDB Cloud cluster.
+title: Quickstart with CockroachDB
+summary: Get started with a free CockroachDB Cloud cluster.
 toc: true
 referral_id: docs_quickstart_free
-filter_category: qs_crdb_cloud
-filter_html: CockroachDB Serverless (beta)
-filter_sort: 1
 docs_area: get_started
 ---
 
-{% include filter-tabs.md %}
+This page shows you how to get started with {{ site.data.products.db }} quickly. You'll create a CockroachDB cluster, and then insert and read some sample data from a sample application.
```

Comment:
The liquid variable is translating to "CockroachDB Cloud", which isn't correct for when you select to work with a local cluster. Can we just use "CockroachDB"?

Reply:
> > _[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12808)_ status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ericharmeling, @ianjevans, and @jseldess)
> 
> _[`cockroachcloud/quickstart.md` line 37 at r5](https://reviewable.io/reviews/cockroachdb/docs/12808#-MtyeHHATZRrYxULdUt4-r5-37:-N0WaI4iDz5PfXD4Gf73:b-h6u5al) ([raw file](https://github.com/cockroachdb/docs/blob/aa68129f21f670d21f1dd3ff13210b4137d62a46/cockroachcloud/quickstart.md#L37)):_
> 
> Previously, ericharmeling (Eric Harmeling) wrote…
> Since the Create SQL User dialog displays automatically, based on the last substep of Step 1, should we consider just merging this all into "Step 1. Create a free cluster"? Or will that not work with other places the include is being used?

I think I'd rather keep the include as is, so I've conditionalized all the step headings and turned this back into a separate step section.

> The liquid variable is translating to "CockroachDB Cloud", which isn't correct for when you select to work with a local cluster. Can we just use "CockroachDB"?

Missed this before. Done.